src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -825,118 +662,44 @@ DistributedObjectMetadataSyncCommandList(void)
-		commandList = list_concat(commandList, sequenceDependencyCommandList);
-	}
 
 	/* command to insert pg_dist_partition entry */
 	char *metadataCommand = DistributionCreateCommand(cacheEntry);
 	commandList = lappend(commandList, metadataCommand);
 
-	/* commands to create the truncate trigger of the table */
-	if (!IsForeignTable(relationId))
-	{
-		char *truncateTriggerCreateCommand = TruncateTriggerCreateCommand(relationId);
-		commandList = lappend(commandList, truncateTriggerCreateCommand);
-	}
-
 	/* commands to insert pg_dist_shard & pg_dist_placement entries */
 	List *shardIntervalList = LoadShardIntervalList(relationId);
 	List *shardMetadataInsertCommandList = ShardListInsertCommand(shardIntervalList);
 	commandList = list_concat(commandList, shardMetadataInsertCommandList);
 
-	if (!tableOwnedByExtension)
-	{
-		/* commands to create foreign key constraints */
-		List *foreignConstraintCommands =
-			GetReferencingForeignConstaintCommands(relationId);
-		commandList = list_concat(commandList, foreignConstraintCommands);
-
-		/* commands to create partitioning hierarchy */
-		if (PartitionTable(relationId))
-		{
-			char *alterTableAttachPartitionCommands =
-				GenerateAlterTableAttachPartitionCommand(relationId);
-			commandList = lappend(commandList, alterTableAttachPartitionCommands);
-		}
-	}
-
 	return commandList;
 }
 
 
 /*
- * MetadataDropCommands returns list of queries that are required to
- * drop all the metadata of the node that are related to clustered tables.
+ * NodeMetadataDropCommands returns list of queries that are required to
+ * drop all the metadata of the node that are not related to clustered tables.
  * The drop metadata snapshot commands includes the following queries:
  *
- * (i)   Query to disable DDL propagation (necessary for (ii)
- * (ii)  Queries that DETACH all partitions of distributed tables
- * (iii) Queries that delete all the rows from pg_dist_node table
- * (iv)  Queries that drop the clustered tables and remove its references from
- *        the pg_dist_partition. Note that distributed relation ids are gathered
- *        from the worker itself to prevent dropping any non-distributed tables
- *        with the same name.
- * (v)   Queries that delete all the rows from pg_dist_shard table referenced by (iv)
- * (vi)  Queries that delete all the rows from pg_dist_placement table
- *        referenced by (v)
- * (vii) Queries that delete all the rows from pg_dist_object table
+ * (i) Queries that delete all the rows from pg_dist_node table
  */
 List *
-MetadataDropCommands(void)
+NodeMetadataDropCommands(void)
 {
 	List *dropSnapshotCommandList = NIL;
-	List *detachPartitionCommandList = DetachPartitionCommandList();
-
-	dropSnapshotCommandList = list_concat(dropSnapshotCommandList,
-										  detachPartitionCommandList);
-
-	/*
-	 * We are re-creating the metadata, so not lose track of the
-	 * sequences by preventing them dropped via DROP TABLE.
-	 */
-	dropSnapshotCommandList =
```

Comment:
We don't seem to use `BREAK_CITUS_TABLE_SEQUENCE_DEPENDENCY_COMMAND` anywhere.

How is this possible that these tests pass? https://github.com/citusdata/citus/pull/5584/files#diff-66dbbcd2698cd91970f40f6c6d793adc2ff30684ea7d172ea8048c200e0ad305